docs/CHANGELOG.rst

```diff
@@ -9,6 +9,10 @@ All notable changes to this project are documented in this file.
 Unreleased
 ==========
 
+Added
+-----
+* Remove threads parameter from cuffquant and cufnorm helper functions
```

Comment:
Move this under "Changed" and put "threads", "cuffquant" and "cuffnorm" in double backticks (``).